about/zarroboogs.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Zarro Boogs Corporation"
+redirect_from:
+- /zbc
+- /about/zbc
+---
+
+<p>Zarro Boogs Corporation (ZBC) is a non-charitable non-profit organization
+(US Tax Code §501(c)(4) status pending IRS approval), and registered as a
+Non-Profit Corporation in the State of Michigan, which means we are required to
+spend all money raised on the needs of the project, and not make a profit on
+it. However, since we are not a charity, any donations will not be tax
+deductible. The Bugzilla trademarks are used under license from the Mozilla
+Foundation. ZBC is not owned by or affiliated with the Mozilla Foundation.</p>
+
+<p>The name "Zarro Boogs Corporation" is a shout-out to the phrase returned by
+Bugzilla when you run a search which returns no results, “Zarro Boogs found.”
+The buggy spelling of “Zero Bugs” being intentional because it’s generally
+believed that there’s no such thing as a project with zero bugs in it, only
+bugs that haven’t yet been reported, thus, saying “Zero Bugs” is, in itself,
+buggy. There’s a nice write-up of this on <a
+href="https://en.wikipedia.org/wiki/Bugzilla#Zarro_Boogs">Wikipedia</a>.</p>
+
+<p>ZBC was incorporated on July 19, 2023. The usage of the Bugzilla trademarks
+were granted by the Mozilla Foundation on August 1, 2023. The formation of the
+corporation was announced to the public on August 26, 2023, which was the 25th
+anniversary of Bugzilla's first public release.</p>
+
+## Board of Directors
+
+The board of directors currently consists of the following people:
+
+{% assign profile = site.developers | where_exp: "item", "item.board_position == 'President'" %}
```

Comment:
I would probably just pick out the profiles where board position isn't empty, and ordered by the board position. That would change the order to President, Secretary, Treasurer but it should be fine I think. Alternatively you could define the array with those names, and iterate over that

Reply:
There's a potential for future additional board members who won't be officers (and thus just "Board Member") so I was trying to future-proof it.  I ended up addressing this by defining an array of the positions in order at the top of the file and then looping through the page variable.

Reply:
Seems like it would be one less step if I could filter on board_position != '' and sort by that array but I couldn't find a way to specify that the sort should be in a specific order...  maybe my search-fu is just weak.